src/models/sensor-definitions.ts

```diff
@@ -658,5 +686,94 @@ export const SensorDefinitions:ISensorDefinitions = {
     "tareable": false,
     "minReading": -35.0,
     "maxReading": 35.0
+  },
+  "g/m³" : {
+    "sensorName": i18n.t("sensor.names.absolute_humidity"),
+    "measurementName": i18n.t("sensor.measurements.absolute_humidity"),
+    "measurementType": "absolute humidity",
+    "tareable": false,
+    "minReading": 0,
+    "maxReading": 100,
+  },
+  "m/s_WS" : {
+    "sensorName": i18n.t("sensor.names.wind_speed"),
+    "measurementName": i18n.t("sensor.measurements.wind_speed"),
+    "measurementType": "wind speed",
+    "tareable": false,
+    "minReading": 0,
+    "maxReading": 100,
+  },
+  "°_WD" : {
+    "sensorName": i18n.t("sensor.names.wind_direction"),
+    "measurementName": i18n.t("sensor.measurements.wind_direction"),
+    "measurementType": "wind direction",
+    "tareable": false,
+    "minReading": 0,
+    "maxReading": 360,
+  },
+  "°C_WC" : {
+    "sensorName": i18n.t("sensor.names.wind_chilll"),
+    "measurementName": i18n.t("sensor.measurements.wind_chilll"),
+    "measurementType": "wind chill",
+    "tareable": false,
+    "minReading": -40,
+    "maxReading":  125,
+  },
+  "°C_HI" : {
+    "sensorName": i18n.t("sensor.names.heat_index"),
+    "measurementName": i18n.t("sensor.measurements.heat_index"),
+    "measurementType": "heat index",
+    "tareable": false,
+    "minReading": -40,
+    "maxReading":  125,
+  },
+  "°C_DP" : {
+    "sensorName": i18n.t("sensor.names.dew_point"),
+    "measurementName": i18n.t("sensor.measurements.dew_point"),
+    "measurementType": "dew_point",
+    "tareable": false,
+    "minReading": -40,
+    "maxReading":  125,
+  },
+  "mbar_SP": {
+    "sensorName": i18n.t("sensor.names.station_pressure"),
+    "measurementName": i18n.t("sensor.measurements.station_pressure"),
+    "measurementType": "station pressure",
+    "tareable": false,
+    "minReading": 0,
+    "maxReading":  1500
+  },
+  "mbar_BP": {
+    "sensorName": i18n.t("sensor.names.barometric_pressure"),
+    "measurementName": i18n.t("sensor.measurements.barometric_pressure"),
+    "measurementType": "barometric pressure",
+    "tareable": false,
+    "minReading": 0,
+    "maxReading":  1500
+  },
+  "m_AL": {
+    "sensorName": i18n.t("sensor.names.altitude"),
+    "measurementName": i18n.t("sensor.measurements.altitude"),
+    "measurementType": "altitude",
+    "tareable": false,
+    "minReading": -300,
+    "maxReading":  10000
   }
 } as const;
+
+export const getSensorUnits = (sensor: any) => {
+  if (Object.keys(SpecialMeasurementUnits).includes(sensor.name)){
+    return SpecialMeasurementUnits[sensor.name];
+  } else {
+    return sensor.unit;
+  }
+}
```

Comment:
I'd move this into sensor-gdx-manager since this lookup is currently specific the gdx sensors.